keycloak_add_users.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+import json
+import requests
+import sys
+import getopt
```

Comment:
Really consider using `click` for anything command line it's a fantastic library and used extensively within the IQE ecosystem.